Libraries/xcassets/Sources/Slot/ColorSpace.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <xcassets/Slot/ColorSpace.h>
+
+#include <cstdlib>
+
+using xcassets::Slot::ColorSpace;
+using xcassets::Slot::ColorSpaces;
+
+ext::optional<ColorSpace> ColorSpaces::
+Parse(std::string const &value)
+{
+    if (value == "sRGB") {
```

Comment:
Having long enums like this is really not the nicest way of handling these mappings :/